scripts/playerCommands.ts

```diff
@@ -687,6 +730,19 @@ Please stop attacking and [lime]build defenses[] first!`
 	//	 }
 	// },
 
+	//this was made with bees in mind
+	overridemap:{
+		args: ["map:map"],
+		description: 'Override the next map in queue.',
+		perm: Perm.admin,
+		handler({args,sender}){
+			Vars.maps.setNextMapOverride(args.map);
+			allCommands.nextmap.data.resetVotes()
```

Comment:
this won't actually work, as it removes the votes, but not the task to end the vote
actually it'll cause a crash
I'll add a function to cancel the vote